tests/unit/models/instrumentation-test.js

```diff
@@ -72,6 +75,35 @@ describe('models/instrumentation.js', function() {
     it('if instrumentation is enabled, monitor', function() {
       process.env.EMBER_CLI_INSTRUMENTATION = '1';
       let monitor = Instrumentation._enableFSMonitorIfInstrumentationEnabled();
+      try {
+        expect(fs.statSync).to.not.equal(originalStatSync);
```

Comment:
This test fails due to `heimdall-fs-monitor` not fully cleaning up after itself. Some module level variables are set in a previous test which cause it be `isEnabled` already. This means when calling `FSMonitor()` a second time, it doesn't re-"attach" the monitor functions around the `fs` functions.

Reply:
okay it looks like this problem in `heimdall-fs-monitor` was actually fixed a while ago.  I've released 0.2.0 to deal with `stop` not cleaning up properly.